plugins/ufm_consumer_plugin/scripts/deinit.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright (C) 2013-2024 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# This software product is a proprietary product of Nvidia Corporation and its affiliates
+# (the "Company") and all right, title, and interest in and to the software
+# product, including all associated intellectual property rights, are and
+# shall remain exclusively with the Company.
+#
+# This software product is governed by the End User License Agreement
+# provided with the software product.
+# @author: Alex Tabachnik
+# @date:   Feb 10, 2024
+#
+
+set -eE
+
+# removing log file
```

Comment:
which log file? 
seems like this script is empty...do we miss any logic on this script?

Reply:
empty file. We do not have special log files to be removed, as Kobi created flow for log files